### DISCUSSION TITLE
目前没有扫一扫功能，后期会考虑添加扫一扫功能么

### DISCUSSION THREAD

**Discussion by LearnMoreAndBetter** [EFPrefix/EFQRCode#119] [Q&A] [answered]

**Comment by ApolloZhu** [comment, marked as answer]
0. https://github.com/EFPrefix/EFQRCode/issues/10
1. 如果是识别图像中的二维码内容，可以使用 [EFQRCodeRecognizer.recognize()](https://efprefix.github.io/EFQRCode/Classes/EFQRCodeRecognizer.html#/c:@M@EFQRCode@objc(cs)EFQRCodeRecognizer(im)recognize)
2. 可以参考其他网上 [Swift](https://www.hackingwithswift.com/example-code/media/how-to-scan-a-qr-code) 或 [SwiftUI](https://www.hackingwithswift.com/books/ios-swiftui/scanning-qr-codes-with-swiftui) 教程实现通过相机识别二维码的功能
